modules/cryoGridTechnical/getActiveLayerDepthAltitude.m

```diff
@@ -1,17 +1,25 @@
-function [ ald_altitude ] = getActiveLayerDepthAltitude( PARA, GRID, T)
+function [ ald_altitude, ald_cT_index] = getActiveLayerDepthAltitude( PARA, GRID, T)
```

Comment:
With the current implementation the name of the function is confusing, beacause it is not the active layer depth anymore (nor was it before, because this would assume permafrost to be present).

Reply:
;-)